modules/router/src/Router.ts

```diff
@@ -290,6 +307,56 @@ export default class ConduitDefaultRouter extends ManagedModule<Config> {
     }
   }
 
+  async registerGrpcProxyRoute(
```

Comment:
proxy routes that come from the grpc-sdk shouldn't be saved in the DB. They should follow the same process as all other routes.